lib/Dialect/SV/Transforms/PrettifyVerilog.cpp

```diff
@@ -104,7 +104,7 @@ void PrettifyVerilogPass::prettifyUnaryOperator(Operation *op) {
   //
   // This is particularly helpful when the operand of the unary op has multiple
   // uses as well.
-  if (op->use_empty() || op->hasOneUse())
```

Comment:
Why was `|| op->hasOneUse()` here? It was blocking moving the "single use" case.

Reply:
The idea was that there is no need to duplicate in this case.  There is a related missing transformation that "sinks" expressions down the region tree.  That should handle the single use-case.

Reply:
In other words, please don't include this change in the patch.